source/includes/steps-fle-convert-to-a-remote-master-key-azure.yaml

```diff
@@ -126,6 +126,31 @@ content: |
                   "clientSecret": "<Azure client secret>"
               }
            }
+
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: chsarp
+
+           var kmsProviders = new Dictionary<string, IReadOnlyDictionary<string, object>>();
+
+           var azureTenantId = Environment.GetEnvironmentVariable("FLE_AZURE_TENANT_ID");
+           var azureClientId = Environment.GetEnvironmentVariable("FLE_AZURE_CLIENT_ID");
+           var azureClientSecret = Environment.GetEnvironmentVariable("FLE_AZURE_CLIENT_SECRET");
+           var azureKmsOptions = new Dictionary<string, object>
+           {
+               { "tenantId", azureTenantId },
+               { "clientId", azureClientId },
+               { "clientSecret", azureClientSecret },
+
```

Comment:
blank line. Check the code in the patch which I sent you, I think I fixed it there